cmd/crc/cmd/delete.go

```diff
@@ -87,3 +93,49 @@ func (s *deleteResult) prettyPrintTo(writer io.Writer) error {
 	}
 	return nil
 }
+
+// TODO: remove it after we do one more upgrade of the podman bundle from 4.4.1
+func removeCRCHostEntriesFromKnownHosts() error {
```

Comment:
This function should be part of `/pkg/crc/machine/delete.go` so that same can be consumed by API endpoints otherwise cmd and API will be different.

Reply:
remove the `TODO` also since it is going to be long lived.